lib/CodeGen/CGException.cpp

```diff
@@ -1750,9 +1752,11 @@ void CodeGenFunction::EmitSEHExceptionCodeSave(CodeGenFunction &ParentCGF,
   //   CONTEXT *ContextRecord;
   // };
   // int exceptioncode = exception_pointers->ExceptionRecord->ExceptionCode;
-  llvm::Type *RecordTy = CGM.Int32Ty->getPointerTo();
+  unsigned DefaultAS = CGM.getTargetCodeGenInfo().getDefaultAS();
+  llvm::Type *RecordTy = CGM.Int32Ty->getPointerTo(DefaultAS);
```

Comment:
Given how often this pattern applies, I wonder if we should add a `CGM.getPointerTo()` method that takes a type and returns a pointer to it in the default AS.